packages/analytics-nextjs/src/context.tsx

```diff
@@ -31,6 +31,10 @@ interface Props {
     plugins?: Plugin[];
     segmentWriteKey?: string;
     plausibleDomain?: string;
+    /**
+     * Skips user consent checks. Use only if you're really brave.
+     */
+    cowboyMode?: boolean;
```

Comment:
Is this really the prop name we want to go with? 😅

Reply:
Well we can name it ‘ignoreConsent’ if you think it’s more professional 😁

Reply:
Well, it's certainly more descriptive 😄